lib/Doctrine/Migrations/Tools/Console/Command/ExecuteCommand.php

```diff
@@ -10,12 +10,13 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
+use function array_map;
 use function getcwd;
 use function is_string;
 use function is_writable;
 
 /**
- * The ExecutCommand class is responsible for executing a single migration version up or down.
+ * The ExecutCommand class is responsible for executing migration versions up or down manually.
```

Comment:
Let's take this as an occasion to fix the typo?

```suggestion
 * The ExecuteCommand class is responsible for executing migration versions up or down manually.
```